experimenter/experimenter/nimbus_ui_new/views.py

```diff
@@ -343,7 +343,22 @@ def form_valid(self, form):
         return response
 
 
+class SaveAndContinueMixin:
+    def form_valid(self, form):
+        response = super().form_valid(form)
+        if (
+            self.request.headers.get("HX-Request")
+            and self.request.POST.get("save_action") == "continue"
+        ):
+            response = HttpResponse()
+            response.headers["HX-Redirect"] = reverse(
+                self.continue_url_name, kwargs={"slug": self.object.slug}
+            )
```

Comment:
everything looks sane, I see where these values come from, but I don't understand what's happening on this line.

Reply:
Ya so we're rewriting the Experimenter UI in a combination of Django templates and [HTMX](https://htmx.org/) which is a handy dandy little library that lets you add some dynamic behaviours without needing to write a ton of JS.  So in this case, we set the [HX-Redirect header](https://htmx.org/headers/hx-redirect/) and HTMX picks it up and redirects the client to the destination URL.

HTMX is a bit weird to get used to at first if you haven't used it, I can give you a more detailed tour if it would help.

Reply:
that clears it up, tyty